### ISSUE TITLE
`indent_style = space` is ignored if `indent_size = unset`

### ISSUE THREAD

**Issue by blue42u** [editorconfig-checker/editorconfig-checker#277]
If the `indent_size` is not set for a file, the check for `indent_style = space` is silently skipped.

Example:
```ini
# .editorconfig
root = true
[*]
indent_style = space
# indent_size = 3
```
```
# example.txt (whitespace made visible)
this line is unindented -- should PASS
····this line is indented with 4 spaces -- should PASS
￫       this line is indented with 1 tab -- should FAIL
￫       ····this line is indented with 1 tab and 4 spaces -- should FAIL
```
```console
$ ec example.txt  # Should fail on the latter 2 lines
$ echo $?
0
```

Note that if the `indent_size = 3` is uncommented from the `.editorconfig` but we disable the indentation size check, the expected errors are reported:
```console
$ ec -disable-indent-size example.txt
example.txt:
	3: Wrong indent style found (tabs instead of spaces)
	4: Wrong indent style found (tabs instead of spaces)

2 errors found
```

**Comment by mstruebing** [comment]
> If the indent_size is not set for a file, the check for indent_style = space is silently skipped.

This is exactly how it is implemented: https://github.com/editorconfig-checker/editorconfig-checker/blob/main/pkg/validation/validators/validators.go#L29 

> Note that if the indent_size = 3 is uncommented from the .editorconfig but we disable the indentation size check, the expected errors are reported:

By looking at the code, the space validator is only executed if it is set to `space`: https://github.com/editorconfig-checker/editorconfig-checker/blob/main/pkg/validation/validators/validators.go#L17C1-L17C1
This means we or https://github.com/editorconfig/editorconfig-core-go is setting this default (I did not find a place in our code by quickly scamming it)
Can you try to specifically set `indent_style` to `unset` in your second example?

The first example is definitely a bug.
